scripts/deploy-community-operator.sh

```diff
@@ -80,12 +79,15 @@ oc patch installplan \
 	--namespace "$TNF_EXAMPLE_CNF_NAMESPACE" \
 	--type merge \
 	--patch '{"spec":{"approved":true}}' \
-	"$( \
+	"$(
 		oc get installplan \
-		--namespace "$TNF_EXAMPLE_CNF_NAMESPACE" |
+			--namespace "$TNF_EXAMPLE_CNF_NAMESPACE" |
 			grep "$COMMUNITY_OPERATOR_NAME" |
-			cut -f 1 -d \  \
-	)" || { printf >&2 'Unable to approve the installation plan.\n'; exit 1;}
+			cut -f 1 -d ' '
+	)" || {
+	printf >&2 'Unable to approve the installation plan.\n'
+	exit 1
+}
 sleep 30
```

Comment:
deployment fails for me here thought not because of this PR. There are a lot of sleep in this file which sometime time out. Makes it difficult to test this PR. Will try to fix it

Reply:
It works for me. The logic should not be changed from `main`. Does the `main` work for you?